executor/common_bsd.h

```diff
@@ -18,3 +18,34 @@ static int do_sandbox_none(void)
 #define do_sandbox_setuid() 0
 #define do_sandbox_namespace() 0
 #endif
+
+#if GOOS_openbsd
+
+#define __syscall syscall
+
+#if SYZ_EXECUTOR || __NR_syz_open_pts
+
+#if defined(__OpenBSD__)
+#include <termios.h>
+#include <util.h>
+#else
+// Needed when compiling on Linux.
```

Comment:
Could somebody explain this to me? Does common_bsd.h need to compile cleanly on Linux? I am thinking of extending this with network packet injection. It might be harder to make cross-platform compatible.